src/Evolution/Systems/CurvedScalarWave/Tags.hpp

```diff
@@ -40,4 +42,46 @@ struct ConstraintGamma2 : db::SimpleTag {
   using type = Scalar<DataVector>;
   static std::string name() noexcept { return "ConstraintGamma2"; }
 };
+
+// @{
+/// \brief Tags corresponding to the characteristic fields of the
+/// scalar-wave system in curved spacetime.
+///
+/// \details For details on how these are defined and computed, \see
+/// CharacteristicSpeedsCompute
+struct VPsi : db::SimpleTag {
+  using type = Scalar<DataVector>;
+};
+template <size_t Dim>
+struct VZero : db::SimpleTag {
+  using type = tnsr::i<DataVector, Dim, Frame::Inertial>;
+};
+struct VPlus : db::SimpleTag {
+  using type = Scalar<DataVector>;
+};
+struct VMinus : db::SimpleTag {
+  using type = Scalar<DataVector>;
+};
+// @}
+
+template <size_t Dim>
+struct CharacteristicSpeeds : db::SimpleTag {
+  using type = std::array<DataVector, 4>;
+  static std::string name() noexcept { return "CharacteristicSpeeds"; }
```

Comment:
I think this and the two tags below also don't need `name()` functions anymore.

Reply:
removed these